repository-data/webfiles/src/main/resources/site/src/scss/components/_api-catalogue.scss

```diff
@@ -58,17 +86,23 @@
 
     & .section-heading:hover,
     & .filter-label:hover {
-      background: $yellow-link-hover;
+      background: $yellow-link-hover !important;
       box-shadow: 0 0 0 4px $yellow-link-hover;
       outline: 1px solid $yellow-link-hover;
-      color: $link-hover-colour-nhs;
+      color: $link-hover-colour-nhs !important;
```

Comment:
are these being overwritten by the .section-heading.selected / .filter-label.selected blocks below? if so maybe they could be rewritten as
```
.section-heading.selected,
.filter-label.selected {
  font-weight: bold;
}

.section-heading.selected:not(:hover) {
  // set colour properties here
}
// same for filter-label
```

up to you whether you feel it's necessary or not, i'd just rather avoid using `!important` if possible :stuck_out_tongue_closed_eyes: